.github/workflows/image.yml

```diff
@@ -37,3 +37,38 @@ jobs:
           --tag ghcr.io/getsentry/taskbroker:${{ github.sha }} \
           "${args[@]}" \
           .
+
+  build-taskworker:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - run: docker login --username '${{ github.actor }}' --password-stdin ghcr.io <<< "$GHCR_TOKEN"
+      env:
+        GHCR_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      if: github.event_name != 'pull_request'
+
+    - run: docker buildx create --driver docker-container --use
+
+    - name: build
+      run: |
+        set -euxo pipefail
+
+        if [ ${{ github.event_name }} = 'push' ]; then
+          args=(
+            --tag ghcr.io/getsentry/taskworker:latest
```

Comment:
Small question: Should this and the other references be `ghcr.io/getsentry/taskbroker:latest` instead or are we calling it taskworker?

Reply:
I'm calling this image `taskworker` to distinguish it from `taskbroker`. Open to suggestions on names though.